js/apps/admin-ui/src/clients/authorization/policy/PolicyDetails.tsx

```diff
@@ -101,6 +104,11 @@ export default function PolicyDetails() {
     ({ policy, policies }) => {
       reset({ ...policy, policies });
       setPolicy(policy);
+      if (policy !== undefined) {
+        if (policy.name === "Default Policy") {
```

Comment:
Is there no better way to detect the default policy? Using the human-readable name seems like an error prone approach. I am worried that this might not work under different locales.

@ssilvert WDYT?

Reply:
I don't think the default policy name is localizable.  But I would need to look at the back end to know for sure.

Reply:
I think it originates from [ResourceServerService.java] (https://github.com/keycloak/keycloak/blob/main/services/src/main/java/org/keycloak/authorization/admin/ResourceServerService.java#L181) 

As it's hardcoded, I think, it should not be an issue.

Reply:
Nice, thanks for checking that. I think I would prefer if the API returned a proper field for it (like `isReadOnly` or somthing). But I think that kind of change is out of scope for this issue.